ch05.adoc

```diff
@@ -211,50 +188,49 @@ If the horizontal spatial coordinates are easting and northing in a map projecti
 
-Where an extended "<gridMappingVariable>: <coordinatesVariable> [<coordinatesVariable>]" entity is defined, then the order of the <coordinatesVariable> references within the definition provides an explicit order for these coordinate value variables, which is used if they are to be combined into individual coordinate tuples.
+Where an extended `<gridMappingVariable>: <coordinatesVariable> [<coordinatesVariable>]` entity is defined, then the order of the `<coordinatesVariable>` references within the definition provides an explicit order for these coordinate value variables, which is used if they are to be combined into individual coordinate tuples.
 
-This order is only significant if crs_wkt is also specified within the referenced grid mapping variable.
-Explicit 'axis order' is important when the grid_mapping_variable contains an attribute crs_wkt as it is mandated by the OGC CRS-WKT standard that coordinate tuples with correct axis order are provided as part of the reference to a Coordinate Reference System.
+This order is only significant if `crs_wkt` is also specified within the referenced grid mapping variable.
+Explicit 'axis order' is important when the __grid mapping variable__ contains an attribute `crs_wkt` as it is mandated by the OGC CRS-WKT standard that coordinate tuples with correct axis order are provided as part of the reference to a Coordinate Reference System.
 
-Using the simple form, where the **`grid_mapping`** attribute is only the name of a grid mapping variable, 2D latitude and longitude coordinates for a projected coordinate reference system use the same geographic coordinate reference system (ellipsoid and prime meridian) as the projection is projected from.
+Using the simple form, where the `grid_mapping` attribute is only the name of a grid mapping variable, 2D latitude and longitude coordinates for a projected coordinate reference system use the same geographic coordinate reference system (ellipsoid and prime meridian) as the projection is projected from.
 
 The `grid_mapping` variable may identify datums (such as the reference ellipsoid, the geoid or the prime meridian) for horizontal or vertical coordinates.
 Therefore a grid mapping variable may be needed when the coordinate variables for a horizontal grid are longitude and latitude.
 The `grid_mapping_name` of `latitude_longitude` should be used in this case.
 
-The expanded form of the **`grid_mapping attribute`** is required if one wants to store coordinate information for more than one coordinate reference system.
-In this case each coordinate or auxiliary coordinate is defined explicitly with respect to no more than one **`grid_mapping`** variable.
+The expanded form of the `grid_mapping` attribute is required if one wants to store coordinate information for more than one coordinate reference system.
```

Comment:
Good spot!

Reply:
bold?

Reply:
Oh, I possibly commented on the wrong line here, but there was a missing closing parenthesis (though on second look, maybe I was misinterpreting somehow...).